members/TXL000482.yaml

```diff
@@ -3,7 +3,7 @@ contact_form:
   method: post
   action: ""
   steps:
-    - visit: "https://senate.texas.gov/memberform.php?auth=jsL%2BLq7RZxMMnLowTCgcKvib1NxSjJKLqdbVvWiMHVI6XzpfOjUlptgR5wozlBFC6aiiugz5M4UHFUH%2FYzXUa%2Fn%2FMSO3Ofk6fDp8OkMcaB%2FJyr%2B%2B4tNYK9zabz0%3D"
+    - visit: "https://senate.texas.gov/memberform.php?auth=p4wIHXNjjLXLIn13LNkZMs6XaNhphVkPWjLPYENBJ2w6XzpfOu4Yys%2FKj5gbGPmVuDpmM2ftzIBcwyLiTQo5dvAynvvXlCk6fDp8OmzYLp3pmXg%2F8A6efqkAx9g%3D&lang=en"
```

Comment:
Is there a reason this can't just go to `https://senate.texas.gov/memberform.php?auth=CFRooBISW%2B824XH6u1kZFY6PX67rZiKvYGxQV%2B8ZUGg6XzpfOmC1%2FtlhdEqAdpSKShJ7r%2B1XAjMNi6P5UzH4N10fecv4TD86fDp8Oq07HU6411IZIdvR2ajQBHw%3D&lang=en` directly? Or just load `https://senate.texas.gov/member.php?d=7&lang=en` and navigate to the selector below?

Reply:
Good point. I edited the URL for the shorter version